docs/topics/developer-tools.rst

```diff
@@ -0,0 +1,248 @@
+.. _topics-developer-tools:
+
+==========================
+Using your browser's Developer Tools for scraping
+==========================
```

Comment:
Travis fails with "Title overline too short." - would you mind increaseing the length of ==== to cover the whole text?

Reply:
Sure thing, I increased the length.